style.css

```
@@ -1687,6 +1687,22 @@ ul.primary-menu {
 	transform: rotate(180deg);
 }
 
+/* Nav submenu expansion on large-viewport touch screens (e.g. tablets) */
+@media (hover: none) and (pointer: coarse) {
```

Another option here would be:

```suggestion
@media (any-pointer: coarse) {
```

This should ensure that devices with both touch screens _and_ trackpads (or mice) would still allow tapping to expand submenus. Many laptops today also feature touch screens.

I'm not an expert on these media queries. I relied on https://medium.com/@ferie/detect-a-touch-device-with-only-css-9f8e30fa1134

---

Done in d5f02f0.